src/bepasty/apis/rest.py

```diff
@@ -15,7 +16,44 @@
 from ..utils.upload import Upload, background_compute_hash
 
 
-class ItemUploadView(MethodView):
+# This wrapper to handle exception of REST api.
+# @blueprint.add_errorhandler decorator can do it though. We have
+# "/lodgeit/" in same blueprint, and didn't find the way to exclude
+# "/lodgeit/" from add_errorhandler to handle exception as web site,
+# not REST api.
```

Comment:
Is this what you mean?
```
This wrappper handles exceptions in the REST api implementation.

The @blueprint.add_errorhandler decorator could do this, but we have
"/lodgeit/" in the same blueprint and there is no way to exclude this from
using the same error handler.
We can't use the same error handler though, because ...
```

Reply:
thanks. fixed as following

```python
# This wrappper handles exceptions in the REST api implementation.
#
# The @blueprint.add_errorhandler decorator could do this, but we have
# "/lodgeit/" in the same blueprint and there is no way to exclude
# from using the same error handler ("/lodgeit/" is not REST api).
```